chrome/content/zotero/fileInterface.js

```diff
@@ -76,6 +76,8 @@ Zotero_File_Exporter.prototype.save = function() {
 			translation.setCollection(this.collection);
 		} else if(this.items) {
 			translation.setItems(this.items);
+		} else {
+			translation.setLibraryID(this.libraryID);
```

Comment:
Let's make sure that `this.libraryID` is set and throw if none of the above are set. Currently, all code sets one of these, but it will probably make debugging a lot easier if we forget at some point. (throw something like "collection, items, or libraryID must be set before calling Zotero_File_Exporter#save()")